hotwax/dxp-components
# Implemented: menu footer navigation component (#126)

closes #126 
### Steps to verify:
- check that this component works well with apps (like job manager)
- proper working of ion-select
- Verify working of menu footer navigation in job manager app raised PR with changes https://github.com/hotwax/job-manager/pull/608
- Review changes of https://github.com/hotwax/job-manager/pull/608
### Description: 

### Additional Info
